16 Lazy Loading/src/index.tsx

```diff
@@ -2,19 +2,36 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Router, Route, IndexRoute, hashHistory } from 'react-router';
 import App from './components/app.tsx';
-import AboutPage from './components/about/aboutPage';
-import MembersPage from './components/members/membersPage';
 import MemberPage from './components/member/memberPage';
 
+const lazyLoadAboutComponent = () => {
+  return {
+      getComponent: (loc, cb)=> {
+        require.ensure([], require => {
+          cb(null, require('./components/about/aboutPage')["default"]);
+        }, 'AboutPage');
+      }
+    }
+};
+
+const lazyLoadMembersComponent = () => {
```

Comment:
Once you are done with that gimme a buzz I think we will be ready for merge request, bells and whistles :)